vault-core-it/vault-core-integration-tests/src/main/java/org/apache/jackrabbit/vault/packaging/integration/UserContentPackageIT.java

```diff
@@ -105,6 +105,15 @@ public void installUserA_Replace_PkgPath() throws RepositoryException, IOExcepti
         installUserA(ImportMode.REPLACE, true, true);
     }
 
+    @Test
```

Comment:
This test is IMHO wrong, as the to be tested package contains invalid filename encoding.